agreement_legal/static/src/js/domain_widget_ext.js

```diff
@@ -52,9 +53,9 @@ odoo.define("agreement_legal.domain_widget_ext", function(require) {
                         partialUse: this.partialUse || false,
                     }
                 );
-                const def = this.domainSelector.prependTo(this.$el);
+                def = this.domainSelector.prependTo(this.$el);
             } else {
-                const def = this.domainSelector.setDomain(value);
+                def = this.domainSelector.setDomain(value);
             }
             // ... then replace the other content (matched records, etc)
             // eslint-disable-next-line no-undef
```

Comment:
It seems like silenting this line was a shoot in the foot. Please remove this comment.

Reply:
I was just realizing the same :smile: Already changed, thanks!